<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主页面</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="BackgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border-image: url(':/UI/bg.png');
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>50</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(':/UI/btn_close_1.png')
}
QPushButton:hover{
   border:10px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>861</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	font:9pt;
	border-radius:5px;
	text-align:center;
	clolor:rgb(255, 255, 255);
	background-color: rgb(255, 255, 255, 0.5);
}
QProgressBar:chunk{
	border-radius:5px;
	background-color:rgb(0, 255, 255)
}
</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowText">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>891</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font : bold;
font-size : 20px;
color:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>正在检查更新</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowText_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>510</y>
     <width>640</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font : bold;
font-size : 20px;
color:rgb(255, 255, 255);
alignment: AlignHCenter;
alignment: AlignVCenter;</string>
   </property>
   <property name="text">
    <string>总说明</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowText_Version">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowText_Website">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>官方网站</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowText_Website_fallback">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>问题反馈</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_Update" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>631</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <widget class="QTextBrowser" name="textBrowser_Msg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>551</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0, 50%);
color:rgb(255, 255, 255);
font : bold;
font-size : 20px;
border-radius:30px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_uuid">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>uuid</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>80</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(':/UI/btn_close_1.png')
}
QPushButton:hover{
	background-color: rgb(255, 0, 0, 50%)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border-image: url(':/UI/company_logo.png');
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>BackgroundLabel</zorder>
  <zorder>progressBar</zorder>
  <zorder>labelShowText</zorder>
  <zorder>labelShowText_2</zorder>
  <zorder>labelShowText_Version</zorder>
  <zorder>labelShowText_Website</zorder>
  <zorder>labelShowText_Website_fallback</zorder>
  <zorder>label_uuid</zorder>
  <zorder>pushButtonClose</zorder>
  <zorder>pushButton</zorder>
  <zorder>labelLogo</zorder>
  <zorder>widget_Update</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
